src/OrchardCore.Modules/OrchardCore.Forms/Views/Items/ValidationRulePart.cshtml

```diff
@@ -3,17 +3,57 @@
 @{
     var type = Model.Value.Type;
     var option = Html.Raw(Model.Value.Option);
+    var validationMessage = Model.Value.ValidationMessage;
     var formElementPart = Model.Value.ContentItem.As<FormElementPart>();
     var elementId = formElementPart.Id;
 }
 
 <script asp-src="~/OrchardCore.Forms/Scripts/validator.min.js" debug-src="~/OrchardCore.Forms/Scripts/validator.js" asp-name="validator" depends-on="jQuery" at="Foot"></script>
 
 <script at="Foot">
```

Comment:
This whole block is repeated for every form field. Rather, only the `validateElementFunction()` call should.